### ISSUE TITLE
Preventing anyone using the app in other countries

### ISSUE THREAD

**Issue by CCarvell** [NERC-CEH/fit-count-app#66]
Is there anything to prevent someone downloading and using the app, registering etc from another country outside of UK (or Cyprus) once launched there? I can't recall if we constrained the location map.
The Home page of the FIT Count website states that counts can be done anywhere.......

**Comment by kazlauskis** [comment]
Yes, we can specify which countries we want to show the app for.

**Comment by kazlauskis** [comment]
Can we close this one?

**Comment by CCarvell** [comment]
@kazlauskis yes we can close it. I assume we didn't specify countries in the end based on above? I think David was happy to be flexible incase this provides a route for other countries to 'try it out'.
